src/ChannelServer/World/Skill.cs

```diff
@@ -55,6 +55,11 @@ public int SpendSp
 			}
 		}
 
+		/// <summary>
+		/// Over Heat Counter
+		/// </summary>
+		public int OverHeatCount { get; set; }
```

Comment:
I opted to go with the name "Overheat" instead of "OverHeat" in the cool down database, since "Over" and "Heat" as separate words don't make a lot of sense to me. We should stick to one or the other, either correcting the Koreans' typo, or not.